DESCRIPTION

```diff
@@ -43,6 +43,7 @@ Imports:
 Suggests: 
     covr,
     datamodelr,
+    DiagrammeR,
```

Comment:
what is DiagrammeR doing in this PR?

Reply:
I added that because in this build https://travis-ci.org/mrc-ide/naomi/builds/610261750#L3125 the vignette failed to build it looks like because of missing DiagrammeR package so included it specifically. Not sure what has changed, I was guessing something about one of the packages being used has changed so it doesn't download the package automatically. Though not sure about that, I could try and take it out and see if we get the same failure.